jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -283,10 +283,25 @@
     - shell: |
         set -x
         shopt -s extglob
+
+        echo $WORKSPACE
+        pwd
+
         # emptying the workspace
         mkdir -p empty
         rsync -r --delete empty/ ./
 
+        cat <<'EOF' > log-parser-plugin-rules.txt
+        start /^(===+> MKCLOUD STEP START: |MKCLOUD step: )/
```

Comment:
No there are not too many sections, it either matches one line of the new style, or one line of the old style, not both.  It's there for backwards compatibility.